### PR TITLE
doc: move cephadm files to its own directory

### PR DESCRIPTION
**Pull request by sebastian-philipp**
My problem was, `doc/mgr/cephadm` outgrew the scope of regular mgr modules. Cephadm is a deployment system and by pure coincidence partially implemented as a mgr module. 

by moving it to somewhere else, we can make `cephadm/` a real project documentation:

* Installation (day 1)
* Adminisration (day 2)
* Drive groups by @jschmid1 

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by sebastian-philipp** [comment]
cc @asettle

**Comment by sebastian-philipp** [comment]
jenkins render docs

**Comment by sebastian-philipp** [comment]
rendering fails with 

```
Traceback (most recent call last):
  File "/home/jenkins-build/build/workspace/ceph-pr-render-docs/build-doc/virtualenv/lib/python3.5/site-packages/sphinx/ext/autodoc/importer.py", line 32, in import_module
    return importlib.import_module(modname)
  File "/home/jenkins-build/build/workspace/ceph-pr-render-docs/build-doc/virtualenv/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/jenkins-build/build/workspace/ceph-pr-render-docs/src/pybind/mgr/orchestrator/__init__.py", line 3, in <module>
    from .module import OrchestratorCli
  File "/home/jenkins-build/build/workspace/ceph-pr-render-docs/src/pybind/mgr/orchestrator/module.py", line 23, in <module>
    from ._interface import OrchestratorClientMixin, DeviceLightLoc, _cli_read_command, \
  File "/home/jenkins-build/build/workspace/ceph-pr-render-docs/src/pybind/mgr/orchestrator/_interface.py", line 393
    self._value = f'_exception: {e}'
                                   ^
SyntaxError: invalid syntax
```
see https://jenkins.ceph.com/job/ceph-pr-render-docs/782/consoleFull#842358957c212b007-e891-4176-9ee7-2f60eca393b7

**Comment by sebastian-philipp** [comment]
ok - ready for review now.

File: doc/cephadm/index.rst [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 ========================
  Installation (cephadm)
 ========================
 
 .. note:: The *cephadm* bootstrap feature is first introduced in Octopus, and is not yet recommended for production deployments.
 
+cephadm is an orchestrator that does not rely
+on a separate system such as Rook or Ansible, but rather manages nodes
+in a cluster by establishing an SSH connection and issuing explicit
+management commands.
```
Comment (asettle) [inline doc/cephadm/index.rst]: I'd consider rephrasing this so the fact is first. IE:

"cephadm manages nodes in a cluster by establishing an SSH connection and issues explicit management commands. It doesnot rely on separate systems such as Rook or Ansible."

Comment (sebastian-philipp) [reply doc/cephadm/index.rst]: fixed

File: doc/cephadm/index.rst [hunk 2]
```diff
@@ -6,10 +6,9 @@
 
 .. note:: The *cephadm* bootstrap feature is first introduced in Octopus, and is not yet recommended for production deployments.
 
-cephadm is an orchestrator that does not rely
-on a separate system such as Rook or Ansible, but rather manages nodes
-in a cluster by establishing an SSH connection and issuing explicit
-management commands.
+cephadm manages nodes in a cluster by establishing an SSH connection
+and issues explicit management commands. It doesnot rely on
```
Comment (asettle) [inline doc/cephadm/index.rst]:
```suggestion
and issues explicit management commands. It does not rely on

**Pull request merged**